src/front-end/front-end-interface/AddGameScreen.cs

```diff
@@ -36,14 +42,16 @@ private void ConfirmAddGameButton_Click(object sender, EventArgs e)
                 imagePath.EndsWith(".jpg") ||
                 imagePath.EndsWith(".jpeg")))
             {
-                imagePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + "\\src\\front-end\\front-end-interface\\Resources\\Logo.png";
+                imagePath = "Resources/Logo.png";
```

Comment:
"/" vs "\" for windows? I see it was used interchangeably?

Reply:
Yes, '/' slash works too.